includes/functions/functions_email.php

```diff
@@ -506,7 +506,9 @@ function zen_mail($to_name, $to_address, $email_subject, $email_text, $from_emai
             if ($ErrorInfo !== '') {
                 $mail_langs = $mail->getTranslations();
                 if (strpos($ErrorInfo, $mail_langs['recipients_failed']) === false) {
-                    trigger_error('Email Error: ' . $ErrorInfo);
+                   if (strpos($ErrorInfo, 'spam content') !== false) { 
```

Comment:
Isn't that going to log _only_ if that content **is** found?

Reply:
No.  If it's false means if it's not there.

Reply:
Right, but you've used  `!== false` which means that (er) it's _not_ not there.  Shouldn't the conditional use `=== false`?

Reply:
@lat9 is correct.

But if we just use `str_contains()` instead, it's even clearer:

```php
if (!str_contains($ErrorInfo, 'spam content')) {
```

----

Examples:
```php
$ErrorInfo = 'Mail contain spam content';

if (!str_contains($ErrorInfo, 'spam content')) {
    echo 'Not spammy, so LOG IT';
    // trigger_error to do the logging
} else {
    // we don't use this ELSE clause, because we just skip logging
    echo 'Spammy, so do NOT log it.';
}

// RESULT: "Spammy, so do NOT log it."
```

Reply:
ARGH I was looking at the wrong line.  Yes, the new PHP8 functions are much easier to read, will switch to that, thanks for catching this.